vendor/bat-native-ads/src/bat/ads/internal/ads/new_tab_page_ads/new_tab_page_ad_unittest.cc

```diff
@@ -227,96 +200,4 @@ TEST_F(BatAdsNewTabPageAdTest, DoNotFireEventIfCreativeInstanceIdWasNotFound) {
-                                             ConfirmationType::kViewed, Now());
-
-  const int ads_per_hour = features::GetMaximumNewTabPageAdsPerHour();
-
-  FireAdEvents(ad_event, ads_per_hour - 1);
-
-  const std::string& uuid = base::GenerateGUID();
-
-  // Act
-  new_tab_page_ad_->FireEvent(uuid, creative_ad.creative_instance_id,
-                              mojom::NewTabPageAdEventType::kViewed);
-
-  // Assert
-  ExpectAdEventCountEquals(ConfirmationType::kViewed, ads_per_hour);
-}
-
-TEST_F(BatAdsNewTabPageAdTest, DoNotFireEventIfExceededAdsPerHourCap) {
-  // Arrange
-  new_tab_page_ads::frequency_capping::ForcePermissionRules();
-
-  const CreativeNewTabPageAdInfo& creative_ad = BuildAndSaveCreativeAd();
-  const AdEventInfo& ad_event = BuildAdEvent(creative_ad, AdType::kNewTabPageAd,
-                                             ConfirmationType::kViewed, Now());
-
-  const int ads_per_hour = features::GetMaximumNewTabPageAdsPerHour();
-
-  FireAdEvents(ad_event, ads_per_hour);
-
-  const std::string& uuid = base::GenerateGUID();
-
-  // Act
-  new_tab_page_ad_->FireEvent(uuid, creative_ad.creative_instance_id,
-                              mojom::NewTabPageAdEventType::kViewed);
-
-  // Assert
-  ExpectAdEventCountEquals(ConfirmationType::kViewed, ads_per_hour);
-}
-
-TEST_F(BatAdsNewTabPageAdTest, FireEventIfNotExceededAdsPerDayCap) {
-  // Arrange
-  new_tab_page_ads::frequency_capping::ForcePermissionRules();
-
-  const CreativeNewTabPageAdInfo& creative_ad = BuildAndSaveCreativeAd();
-  const AdEventInfo& ad_event = BuildAdEvent(creative_ad, AdType::kNewTabPageAd,
-                                             ConfirmationType::kViewed, Now());
-
-  const int ads_per_day = features::GetMaximumNewTabPageAdsPerDay();
-
-  FireAdEvents(ad_event, ads_per_day - 1);
-
-  AdvanceClock(base::Hours(1));
-
-  const std::string& uuid = base::GenerateGUID();
-
-  // Act
-  new_tab_page_ad_->FireEvent(uuid, creative_ad.creative_instance_id,
-                              mojom::NewTabPageAdEventType::kViewed);
-
-  // Assert
-  ExpectAdEventCountEquals(ConfirmationType::kViewed, ads_per_day);
-}
-
-TEST_F(BatAdsNewTabPageAdTest, DoNotFireEventIfExceededAdsPerDayCap) {
-  // Arrange
-  new_tab_page_ads::frequency_capping::ForcePermissionRules();
-
-  const CreativeNewTabPageAdInfo& creative_ad = BuildAndSaveCreativeAd();
-  const AdEventInfo& ad_event = BuildAdEvent(creative_ad, AdType::kNewTabPageAd,
-                                             ConfirmationType::kViewed, Now());
-
-  const int ads_per_day = features::GetMaximumNewTabPageAdsPerDay();
-
-  FireAdEvents(ad_event, ads_per_day);
-
-  AdvanceClock(base::Hours(1));
-
-  const std::string& uuid = base::GenerateGUID();
-
-  // Act
-  new_tab_page_ad_->FireEvent(uuid, creative_ad.creative_instance_id,
-                              mojom::NewTabPageAdEventType::kViewed);
-
-  // Assert
-  ExpectAdEventCountEquals(ConfirmationType::kViewed, ads_per_day);
-}
-
```

Comment:
How come these tests were removed? Thanks

Reply:
Moved these tests to a separate file: [new_tab_page_ad_if_ads_disabled_unittest.cc](https://github.com/brave/brave-core/pull/11945/files#diff-6982bf23f496560ee920b6c2550eae466aaba9a5042982abd56169e5e9b53272)

These tests are running with Brave Ads preference disabled.